policyengine_us/variables/gov/states/ca/tax/income/credits/foster_youth/ca_foster_youth_tax_credit.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class ca_foster_youth_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "California foster youth tax credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "ca_eitc_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ca.tax.income.credits.foster_youth
+        person = tax_unit.members
+        age = person("age", period)
+        eligible_person = person(
+            "ca_foster_youth_tax_credit_eligible_person", period
+        )
+        base_credit = p.amount.calc(age) * eligible_person
+        total_base_credit = tax_unit.sum(base_credit)
```

Comment:
apply the tax_unti.sum after we reduce each individual credit amount (base_credit)